### PR TITLE
Bugfix/Font-wrap-wrong

### PR DESCRIPTION
**Pull request by JucanAndreiDaniel**
Fix glyph metrics to have the glyphs with the right orientation and fix all labels from main menu

# To test
1. Open main menu and check if text looks the same 
2. Create a label with a lot of text so that it wraps and check if text orientation is fine

### CODE REVIEW

**Comment by JaXt0r** [comment]
I tested your feature. Is this the intended outcome? If you can share a before - after image, then I can test better.
image

File: Assets/GothicVR/Scripts/UI/FontManager.cs [hunk 1]
```diff
@@ -49,13 +49,14 @@ public TMP_SpriteAsset LoadFont(string fontName)
...
-                        horizontalBearingY = h,
+                        height = -h,
+                        horizontalBearingY = 0,
                         horizontalBearingX = 0,
                         horizontalAdvance = w
                     },
                     index = (uint)i,
-                    sprite = newSprite
+                    sprite = newSprite,
+                    scale = -1
```
Comment (JaXt0r) [inline Assets/GothicVR/Scripts/UI/FontManager.cs:59]: Why a scale of -1? Sounds wrong tbh.

File: Assets/GothicVR/Resources/Prefabs/UI Element Prefabs/MainMenu.prefab
Comment (JaXt0r) [inline Assets/GothicVR/Resources/Prefabs/UI Element Prefabs/MainMenu.prefab:1]: Hm. Seems you flip about every text element. Why?

**Review by JaXt0r — CHANGES_REQUESTED** [review]

**Comment by JucanAndreiDaniel** [comment]
> I tested your feature. Is this the intended outcome? If you can share a before - after image, then I can test better. image

This is intended


before
image

after

image


these are taken from the discord where i posted what was the issue

**Review by JaXt0r — APPROVED** [review]

**Pull request merged**